tests/console/validate_raid_encrypt_home.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Simple RAID partitioning layout validation for encrypted home
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use Mojo::JSON qw(decode_json);
+
+sub run {
+    select_console 'root-console';
+
+    # To avoid the regression of bsc#1224393, need ensure that the crypt /home
+    # partition is under 'md1' via comparing with the partitioning output of md1.
```

Comment:
Please remove this comment and add briefly the bug reference on the description of the test.
'md1' doesn't need to be mention, should be obvious by the code, I will try to comment on it.

Reply:
Updated.